clients/outline-client-windows.exe

```diff
@@ -0,0 +1 @@
+placeholder
```

Comment:
Right now we simply call it `Outline.exe`. Note that the platform can identified by the filename extension.

I realise now that what we generate is an **installer**. So, we could call it...`Outline_Installer.exe`?